### PR TITLE
FEATURE: Refactor marketplace templates

### PR DESCRIPTION
**Pull request by Sebobo**
Also fix some problems with syncing and template issues.

### CODE REVIEW

**Review by mhsdesign — APPROVED** [review]
Sieht koscher aus

File: DistributionPackages/Neos.MarketPlace/Resources/Private/Fusion/Components/VendorRenderer.fusion [hunk 1]
```diff
@@ -0,0 +1,13 @@
+prototype(Neos.MarketPlace:VendorRenderer) < prototype(Neos.Fusion:Component) {
+    title = ${q(node).property('title')}
+    packages = ${q(node).children('[instanceof Neos.MarketPlace:Package]')}
+
+    renderer = afx`
+        <article class="market-block">
+            <h1>{title}</h1>
```
[the remark this answers is not in the document]
Comment (crydotsnake) [reply DistributionPackages/Neos.MarketPlace/Resources/Private/Fusion/Components/VendorRenderer.fusion:7]: I know i'm too late. But shouldnt this be: `props.title`?

[the remark this answers is not in the document]
Comment (Sebobo) [reply DistributionPackages/Neos.MarketPlace/Resources/Private/Fusion/Components/VendorRenderer.fusion:7]: Yes thx!

[the remark this answers is not in the document]
Comment (crydotsnake) [reply DistributionPackages/Neos.MarketPlace/Resources/Private/Fusion/Components/VendorRenderer.fusion:7]: Np

**Pull request merged**